phabricator.py

```diff
@@ -61,7 +80,8 @@ def run(self):
 
         # Run `arc browse` and dump the output to the console
         browse_path = '{0}:{1}'.format(filename, lines)
-        arc_args = [settings.get('arc_path', 'arc'), 'browse', browse_path, '--branch', escaped_branch]
+        arc_args = [
+            settings.get('arc_path', 'arc'), 'browse', browse_path, '--branch', escaped_branch]
```

Comment:
Yeah, I just had a linter running and ended up fixing all lint errors. Besides this repo seems kinda dead so I just squashed everything into a single PR. :(